AndelaOSP/andela-societies-frontend
# Resolves #56137968 - Add VerifyActivities page

#### Description:
Add verify Activities page. Add ability to switch between pages by clicking links on the sidebar.
Additionally:
- Refactor routing code. All routes are defined in one place.
- Refactor page component code. All pages are wrapped in a `<Page />` component.

#### How to test:
- Checkout to the `ft-create-myactivities-page-156137968` branch.
- Click the VerifyActivities link in the sidebar. The verify activities page should be visible and the title should reflect the change. You should be able to go back to the My Activities page.
